src/NavbarToggler.js

```diff
@@ -15,21 +15,16 @@ const propTypes = {
   children: PropTypes.node,
 };
 
-const defaultProps = {
-  tag: 'button',
-  type: 'button',
-};
-
 function NavbarToggler(props) {
-  const { className, cssModule, children, tag: Tag, ...attributes } = props;
+  const { className, cssModule, children, tag: Tag = 'button', ...attributes } = props;
 
   const classes = mapToCssModules(
     classNames(className, 'navbar-toggler'),
     cssModule,
   );
 
   return (
-    <Tag aria-label="Toggle navigation" {...attributes} className={classes}>
+    <Tag aria-label="Toggle navigation" {...{type: 'button',...attributes}} className={classes}>
```

Comment:
Do we need to do `{...{type: 'button',...attributes}}`? Shouldn't just `{type: 'button',...attributes}` work?

Reply:
`{type: 'button',...attributes}` defaults `type` if it's not present in `attributes`. The wrapping `{...obj}` spreads the resulting object to be rendered as html attributes